scipy/signal/ltisys.py

```diff
@@ -445,6 +423,13 @@ def impulse(self, X0=None, T=None, N=None):
         """
         return impulse(self, X0=X0, T=T, N=N)
 
+    def impulse2(self, X0=None, T=None, N=None, **kwargs):
```

Comment:
From reading the PR it sounds like the `*2` functionality will be moved into `impulse` and `step` and then the `*2` functions will be deprecated. @rgommers do you think these classes should have both `impluse` and `impulse2` for now?